doc/nrf/known_issues.rst

```diff
@@ -326,6 +326,22 @@ Bluetooth LE
 
 .. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
 
+NCSDK-8224: Security changed and pairing failed callbacks are not always called
+  The pairing failed and security changed callbacks are not called when disconnected during the pairing procedure or required security was not met.
+
+  **Workaround:** Application should use the disconnected callback in addition to handle pairing failed.
+
+.. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+NCSDK-8223: GATT requests may deadlock RX thread
+  GATT requests may deadlock the RX thread when all TX buffers are taken by GATT requests and the RX thread tries to allocate a TX buffer for a response.
+  This is deadlocked because only the RX thread releases the TX buffers for the GATT requests.
+  The deadlock is resolved by a 30 second timeout but the ATT bearer cannot transmit without re-connecting.
```

Comment:
```suggestion
NCSDK-8223: GATT requests might deadlock RX thread
  GATT requests might deadlock the RX thread when all TX buffers are taken by GATT requests and the RX thread tries to allocate a TX buffer for a response.
  This causes a deadlock because only the RX thread releases the TX buffers for the GATT requests.
  The deadlock is resolved by a 30 second time-out, but the ATT bearer cannot transmit without reconnecting.
```